src/storage/txn/sched_pool.rs

```diff
@@ -109,6 +109,7 @@ struct PriorityQueue {
 impl PriorityQueue {
     fn spawn(
         &self,
+        request_source: &str,
```

Comment:
why not put it in task metadata

Reply:
Not easy to do so, and `request_source` is only used to spawn the task